src/core/distributions/phylogenetics/substitution/AbstractPhyloCTMCSiteHomogeneous.h

```diff
@@ -2654,7 +2649,10 @@ void RevBayesCore::AbstractPhyloCTMCSiteHomogeneous<charType>::resizeLikelihoodV
         {
             partialLikelihoods[i] = 0.0;
         }
-
+        for(auto&& dirty_node: dirty_nodes)
+        {
+            dirty_node = true;
+        }
```

Comment:
Make sure to mark all the partial likelihoods as invalid, since we just zeroed them all!